packages/markdown/processor/options.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
What's this for? We're not planning on exposing these options to the consumer are we?

Reply:
I'm not sure—I'd been considering exposing them, if only as defaults. But if that gives you pause I'm inclined to say we can bake them in to the Unified processor itself, but a lot of them are critical to correctly rendering our flavor of markdown. (For example, without GFM turned on code blocks won't render properly.)